etsd/core/templates/core/edit_authority_data.html

```diff
@@ -0,0 +1,32 @@
+{% extends "site_base.html" %}
+{% load render_table from django_tables2 %}
+{% load crispy_forms_tags %}
+{% load i18n static %}
+
+
+{% block head_title %}{% blocktrans %}Edit Authority Data{% endblocktrans %}{% endblock %}
+{% block page_title %}{% blocktrans %}Edit Authority Data{% endblocktrans %}{% endblock %}
+{% block page_content %}
+
+<div class='alert alert-info'>
+  {% blocktrans %}
+  <b>Below are your Authority's active users. </b><br>
```

Comment:
Since this view may also be used by admins you may want to make the text more generic (i.e these are the data of the authority X)